pkg/configuration/config_manager/config_handler.go

```diff
@@ -169,37 +171,77 @@ type shellCommandHandler struct {
 	downwardAPIMountPoint []string
 	downwardAPIHandler    map[string]ConfigHandler
 
-	backupPath string
-	configMeta *ConfigSpecInfo
-	filter     regexFilter
+	backupPath    string
+	configMeta    *ConfigSpecInfo
+	filter        regexFilter
+	isBatchReload bool
 }
 
 func (s *shellCommandHandler) OnlineUpdate(ctx context.Context, name string, updatedParams map[string]string) error {
 	logger.V(1).Info(fmt.Sprintf("online update[%v]", updatedParams))
 	logger.Info(fmt.Sprintf("updated parameters: %v", updatedParams))
 	args := make([]string, len(s.arg))
-	return s.execHandler(ctx, updatedParams, args)
+	copy(args, s.arg)
+	_, err := s.execHandler(ctx, updatedParams, args)
+	return err
 }
 
-func (s *shellCommandHandler) execHandler(ctx context.Context, updatedParams map[string]string, args []string) error {
-	commonHandle := func(args []string) error {
+func (s *shellCommandHandler) execHandler(ctx context.Context, updatedParams map[string]string, args []string) ([]string, error) {
+	if s.isBatchReload {
```

Comment:
This if statement branch code is too long and should be refactor into a separate function.

e.g:
```
if s.isBatchReload {
    return xxxxx
}
```

Reply:
Fixed in bbf86cdffc028e83746c42b7fa33d25479c37ec0